diff -aurN sane-backends-1.0.23.orig/tools/sane-backends.pc.in sane-backends-1.0.23/tools/sane-backends.pc.in
--- sane-backends-1.0.23.orig/tools/sane-backends.pc.in	2012-11-05 10:23:40.712546533 +0100
+++ sane-backends-1.0.23/tools/sane-backends.pc.in	2012-11-05 10:24:05.286461911 +0100
@@ -2,7 +2,6 @@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
-ldflags=@LDFLAGS@ @GPHOTO2_LDFLAGS@
 libs=@LIBS@ @DL_LIBS@ @LIBV4L_LIBS@ @MATH_LIB@ @TIFF_LIBS@ @JPEG_LIBS@ @GPHOTO2_LIBS@ @SOCKET_LIBS@ @AVAHI_LIBS@ @USB_LIBS@ @SCSI_LIBS@ @RESMGR_LIBS@
 
 Name: SANE Backends
@@ -10,5 +9,5 @@
 Version: @VERSION@
 Requires:
 Libs: -L${libdir} -lsane
-Libs.private: ${ldflags} ${libs}
+Libs.private: ${libs}
 Cflags: -I${includedir}
